joomla/joomla-cms
# [4.0] Set default Site and Admin languages in installer

Alternative Pull Request for Draft https://github.com/joomla/joomla-cms/pull/27951.

### Summary of Changes
Adding back the ability to set  Site and Admin languages in the installer.


### Testing Instructions

1. Apply this patch. You do **not** need to run `npm ci` or `composer i`.
2. Start a new installation.
3. Install an extra language
4. Make sure, that can set default frontend language and default administrator language, after you had installed the extra language.
![Joomla Installer](https://user-images.githubusercontent.com/9974686/76572955-c8dfe600-64bb-11ea-992c-fff8e979b26a.png)
5. Change the default language for the backend and frontend and make sure that the languages are used correctly.


### Expected result

People who don't speak English as a native language are able to set the admin default language before they start to configure things.

### Actual result

It is not possible to set a default language during installation.


### Documentation Changes Required

Not sure.
